src/ott/utils.py

```diff
@@ -59,3 +60,56 @@ def is_jax_array(obj: Any) -> bool:
     # https://jax.readthedocs.io/en/latest/jax_array_migration.html
     return isinstance(obj, (jax.Array, jnp.DeviceArray))
   return isinstance(obj, jnp.DeviceArray)
+
+
+def default_progress_fn(
```

Comment:
Currently, this is not visible in the online docs and it would be great to have it there! Can you add `docs/utils.rst` (see, e.g., `docs/tools.rst`) and include this function +  add a reference to `docs/utils.rst` in `docs/index.rst`?